<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="26258" started-at="2012-04-28T13:02:52Z" finished-at="2012-04-28T13:03:18Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="26258" started-at="2012-04-28T13:02:52Z" finished-at="2012-04-28T13:03:18Z">
      <class name="tests.AdminTests">
        <test-method status="PASS" signature="setUp(java.lang.String)[pri:0, instance:tests.AdminTests@2d58f9d3]" name="setUp" is-config="true" duration-ms="27" started-at="2012-04-28T13:02:52Z" finished-at="2012-04-28T13:02:52Z">
          <params>
            <param index="0">
              <value is-null="true"/>
            </param>
          </params>
        </test-method>
        <test-method status="FAIL" signature="adminCanLogin()[pri:0, instance:tests.AdminTests@2d58f9d3]" name="adminCanLogin" duration-ms="26101" started-at="2012-04-28T13:02:52Z" finished-at="2012-04-28T13:03:18Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.InstantiationException: pageObjects.LoginPage]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.InstantiationException: pageObjects.LoginPage
	at org.openqa.selenium.support.PageFactory.instantiatePage(PageFactory.java:137)
	at org.openqa.selenium.support.PageFactory.initElements(PageFactory.java:67)
	at pageObjects.LoginPage.open(LoginPage.java:31)
	at frameWork.AdminActions.loginAsAdmin(AdminActions.java:11)
	at tests.AdminTests.adminCanLogin(AdminTests.java:14)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:691)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:883)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1208)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:758)
	at org.testng.TestRunner.run(TestRunner.java:613)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:87)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1142)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1067)
	at org.testng.TestNG.run(TestNG.java:979)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:109)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:202)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:173)
Caused by: java.lang.InstantiationException: pageObjects.LoginPage
	at java.lang.Class.newInstance0(Unknown Source)
	at java.lang.Class.newInstance(Unknown Source)
	at org.openqa.selenium.support.PageFactory.instantiatePage(PageFactory.java:134)
	... 28 more
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="tearDown()[pri:0, instance:tests.AdminTests@2d58f9d3]" name="tearDown" is-config="true" duration-ms="118" started-at="2012-04-28T13:03:18Z" finished-at="2012-04-28T13:03:18Z">
        </test-method>
      </class>
    </test>
  </suite>
</testng-results>
